src/js/legend.js

```diff
@@ -258,4 +323,11 @@ https://raw.githubusercontent.com/fluid-project/chartAuthoring/master/LICENSE.tx
         return colorScale.range();
     };
 
+    floe.chartAuthoring.pieChart.legend.getDisplayValueFromTemplate = function(that, template, d) {
```

Comment:
We probably can pass in ```that.options.legendOptions.legendPercentageDigits``` and ```that.model.total.value``` as function arguments to reduce the dependency on ```that```.